run.py

```diff
@@ -0,0 +1,19 @@
+from scripts.data import load_params
+from scripts.data import load_raw
+from scripts.data import write_annotate
```

Comment:
I see no issue with the structure of the project. However, with this commit, this script will not run. I was trying to test it, but the commit does not include any of these files. Besides that, I see no issue with run.py. I like the change of a ChainMap instead of a simple unordered dict.

Reply:
Sounds good, thanks for the feedback. Closing & repulling.